config/routes.rb

```diff
@@ -64,6 +64,7 @@
 
     resources :application_users, only: [:index] do
       collection do
+        get 'username/:username', action: 'username'
```

Comment:
I agree with @Dantemss.  I like the `../find/` syntax, but don't like that you can't discern between a `id/uid/username` search.  I've just implemented a `../find/username/jpslav` endpoint.  We can add additional finders later if/when we need them.